RecoBTag/Skimming/src/BTagSkimMC.cc

```diff
@@ -91,6 +91,10 @@ void BTagSkimMC::globalEndJob(const BTagSkimMCCount::Counters* count) {
                                  << std::endl;
 }
 
-#include "FWCore/Framework/interface/MakerMacros.h"
+void BTagSkimMC::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
```

Comment:
Sorry, but what's the point of this?
This does not provide any validation nor defaults for any of the actual configuration parameters?

Reply:
@mmusich I've changed most of them to `UntrackedParameter` except for the `InputTag`s so that the default values are set in the constructor itself. The relevant python `cfi` files are modified accordingly.

Reply:
In my limited experience, I don't remember encountering much difference in run time or memory consumption with `Untracked` parameters in the cc files. Can you please elaborate may be with some reference ?